week1/week1-homework.py

```diff
@@ -0,0 +1,72 @@
+# Home work for week1
+
+# 1. Print name and age
+name = 'Gebremedhin'
+age = 34
+print(f'My Name: {name}')
+print(f'My Age: {age}')
+
+# 2.  list of your 5 favorite movies
+favorite_movies = ['Spider man', 'Iron man', 'Salt', 'Titanic', 'Prison break']
+
+# 3. Displaying the first and last colors from the following list
+color_list = ["Red", "Green", "White", "Black"]
+first_and_last_color = []
+selected_color_indexes = [0, (len(color_list) - 1)]
+for index in selected_color_indexes:
+    first_and_last_color.append(color_list[index])
+# print(first_and_last_color)
+
+# .4 Adding a key to a dictionary
+students_mobile_no = {'Alex': '123-456-23', 'Anna': '24-985-12'}
+students_mobile_no['Jhon'] = '44-435-90'
+# print(students_mobile_no)
+
+# 5. Calculating BMI- body mass index
+weight = input('Enter your weight in kg: >  ')
+height = input('Enter your height in m: > ')
+bmi = float(weight) / (float(height) ** 2)
```

Comment:
Good job! Mathematically `power to` (`**`) come before multiplication `*` and division `/`, so no need for brackets in the denominator:
```python
bmi = float(weight) / float(height) ** 2
```